src/components/ReturnsCalculator.jsx

```diff
@@ -0,0 +1,244 @@
+				stashIdTruncated,
+				name,
+				commission,
+				totalStake,
+				poolReward,
+				noOfNominators
+			} = validator;
+			const userStakeFraction =
+				suggPromptsAmount / (suggPromptsAmount + totalStake);
+			const dailyEarning = userStakeFraction * poolReward * ERA_PER_DAY;
+			return {
+				noOfNominators,
+				stashId,
+				stashIdTruncated,
+				name,
+				commission: `${parseFloat(commission)}%`,
+				dailyEarning: isNaN(dailyEarning)
+					? "Not enough data"
+					: `${dailyEarning.toPrecision(10)} KSM`,
+				dailyEarningPrecise: isNaN(dailyEarning) ? 0 : dailyEarning
+			};
+		});
+
+		//handle top 16 suggested validaors
+		data.sort((a, b) => b.dailyEarningPrecise - a.dailyEarningPrecise);
+		const top16data = [...data.slice(0, 16)];
+		//console.log("table data of top 16 val - ", top16data);
+		
+		//pass props to parent App.jsx
+		const propsData = {};
+		if (top16data.length > 0) {
+			// eslint-disable-next-line no-unused-vars
+			const expectedEarning = top16data.reduce((a, b) => ({
+				dailyEarningPrecise: a.dailyEarningPrecise + b.dailyEarningPrecise
+			}));
+			//console.log("expected earning of top 16 val - ", expectedEarning);
+			propsData.expectedReturns = expectedEarning.dailyEarningPrecise;
+			setExpectedReturns(expectedEarning.dailyEarningPrecise);
+		}
+		propsData.validatorsList = top16data;
+		propsData.budget = stakeInput;
+		props.onEvent(propsData);
+		
+		if (apiConnected) setIsLoaded(true);
+	}
+
+	React.useEffect(() => {
+		setValidatorsList(props.validatorData);
+	}, [props]);
+
+	if (errorState) {
+		return <ErrorMessage />;
+	}
+
+	function calculate() {
+		suggPrompts();
+	}
+
+	const onRiskChange = value => {
+		console.log(value);
+		if (value <= 30) {
+			setSliderBG("green.300");
+		} else if (value > 70) {
+			setSliderBG("red.400");
+		} else {
+			setSliderBG("yellow.300");
+		}
+	};
+
+	return (
+		<React.Fragment>
+			<Helmet>
+				<title>YieldScan - Returns Calculator</title>
+				<meta name='description' content='Validator key stats' />
+			</Helmet>
+			<LogEvent eventType='Returns calculator view' />
+			<Route exact path='/returns-calculator'>
+				<Heading mt={16} mb={12} 
+					ml={"calc(15% - 2rem)" // 992px upwards
+					}>
+					Calculate your returns
+				</Heading>
+				<Flex alignItems='center' flexWrap='wrap' justify="center">
+					<Box minWidth='288px' mr={8}>
+						<Box
+							color='gray.500'
+							fontWeight='semibold'
+							letterSpacing='wide'
+							fontSize='md'
+							mb={8}
+						>
+							<Flex flexWrap='wrap'>
+								<Flex direction='column' mr={8} mb={4}>
+									<Text mb={2}>I want to spend</Text>
+									<Input
+										placeholder='Enter your budget'
+										variant='filled'
+										color={textColor[props.colorMode]}
+										type='number'
+										min='0'
+										step='0.000000000001'
+										max='999999999999999'
+										value={stakeInput}
+										textAlign='center'
+										rounded='40px'
+										minWidth='299px'
```

Comment:
Remove minWidth -> causing lack of whitespace in mobile view

Reply:
It's been removed.